src/test/java/seedu/address/model/person/TissuetypeTest.java

```diff
@@ -0,0 +1,65 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class TissuetypeTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Tissuetype(null));
+    }
+
+    @Test
+    public void constructor_invalidTissuetype_throwsIllegalArgumentException() {
+        String invalidTissuetype = "";
+        assertThrows(IllegalArgumentException.class, () -> new Tissuetype(invalidTissuetype));
+    }
+
+    @Test
+    public void isValidTissuetype() {
+        // null tissuetype
+        assertThrows(NullPointerException.class, () -> Tissuetype.isValidTissuetype(null));
+
+        // invalid tissuetype
+        assertFalse(Tissuetype.isValidTissuetype("")); // empty string
+        assertFalse(Tissuetype.isValidTissuetype(" ")); // spaces only
+        assertFalse(Tissuetype.isValidTissuetype("a,b,c,d,e,f")); // wrong tissuetype
+        assertFalse(Tissuetype.isValidTissuetype("1")); // single numeric
+        assertFalse(Tissuetype.isValidTissuetype("1,,2,,3,,4,,5,,6")); // wrong usage of commas
+        assertFalse(Tissuetype.isValidTissuetype("1 2 3 4 5 6")); // spaces within tissuetype
+
+        // valid tissuetype
+        assertTrue(Tissuetype.isValidTissuetype("1,2,3,4,5,6")); // exactly 6 tissuetype
```

Comment:
Could you include test cases for tissue types > 12? Eg 13, 14.

Reply:
yes, line 45

Reply:
Line 45 tests the `toString()` method, not the `isValidTissueType()` method. Could you add test cases which checks for numbers that are not accepted as valid tissue types?

Reply:
yes done